llm_inference.py

```diff
@@ -238,19 +263,23 @@ class InferenceArguments:
 
 
 class LLM(object):
-    """Wrapper class for HuggingFace LLMs."""
-    def __init__(self, args: InferenceArguments) -> None:
+
+    def __init__(self, args: InferenceArguments, is_encoder_decoder: bool = False):
```

Comment:
This is fine, but we could also just reference the argument in `self.args.is_encoder_decoder` to avoid passing it separately.